README.md

```diff
@@ -157,8 +157,24 @@ Example of such app.config:
 ].
 ```
 
+### VPC endpoints
+If you want to utilise AZ affinity for VPC endpoints you can configure those in application config via:
+```erlang
+{erlcloud, [
+    {services_vpc_endpoints, [
+        {<<"sqs">>, [<<"myAZ1.sqs-dns.amazonaws.com">>, <<"myAZ2.sqs-dns.amazonaws.com">>]},
+        {<<"kinesis">>, {env, "KINESIS_VPC_ENDPOINTS"}}
+    ]}
+]}
+```
+Two options supported:
+ - explicit lists of route53 AZ endpoints
+ - OS environment variable, comes handy for ECS deployments.
```

Comment:
We should note in the docs that the OS env variable is parsed as a CSV of VPC endpoints.